app/actions.js

```diff
@@ -167,8 +167,7 @@ const enrichMessage = (message) => {
 }
 
 export const getMessages = ({ addr, channel, amount }, callback) => dispatch => {
-  client.focusCabal(addr)
```

Comment:
this was a bad idea (by me :sweat_smile: ), since it changes the cabal on new messages. Hence, fixing it.